### PR TITLE
documentation/80

### LINKED ISSUE #80: Update git-flow doc
_From @n8rzz on November 21, 2016 0:37_

_Copied from original issue: n8rzz/atc#129_

### PR DESCRIPTION
**Pull request by erikquinn**
Resolves #80.

Update git flow documentation.

@n8rzz Started this, not sure entirely what you wanted to do, but I started it out with a flow chart at least.

### CODE REVIEW

File: documentation/git-flow-process.md [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
-
-- **feature/XX** - new features or refactor work. should have a corresponding github issue where the `XX` is the issue number. features should only ever branch off `develop`. may contain code that is under active development. may have failing tests or non-working pieces. should stay current with `develop` and should be conflict free when submitting a PR. developer may open a [WIP] PR for visibility, though not required.
+```
+         +------------------------+-------------------------+-----------------------------------+
+         |      DEVELOPMENT       |         TESTING         |          INITIALIZATION           |
+         +------------------------+-------------------------+-----------------------------------+
+         |                        |                         |              hotfix    hotfix     |
+         |                        |                         |              o-o-o-o   o-o-o-o    |
+         |                        |                         |             /       \ /       \   |
+ master  +------------------------+-------------------------+------------o---------o---------o--+--→ master
```
Comment (n8rzz) [inline documentation/git-flow-process.md]: what a pretty diagram!

I hate to make you change it, but release should be between master and develop 😬

Comment (erikquinn) [reply documentation/git-flow-process.md]: ugh. yeah, that would make sense. :sob:

Comment (n8rzz) [reply documentation/git-flow-process.md]: why not just reference the diagram in the `nvie` article?

Comment (erikquinn) [reply documentation/git-flow-process.md]: done!

**Pull request merged**